databases/week3/DataBase3.sql

```diff
@@ -0,0 +1,158 @@
+
+CREATE DATABASE HWdataWeek3;
+USE HWdataWeek3;
+
+CREATE TABLE Meal (
+ id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+ title VARCHAR(255) UNIQUE,
+ description_meal TEXT,
+ location_meal VARCHAR(255),
+ when_created DATETIME,
+ max_reservations INT, 
+ price DECIMAL(10,2) NOT NULL,
+ created_date DATE NOT NULL
+);
+
+CREATE TABLE Reservation (
+ id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+ number_of_guests INT NOT NULL,
+ meal_id INT NOT NULL,
+ created_date DATE,
+ contact_phonenumber VARCHAR(255),
+ contact_name VARCHAR(50) NOT NULL,
+ contact_email VARCHAR(255),
+ FOREIGN KEY (meal_id) REFERENCES Meal(id)
+);
+
+CREATE TABLE Review (
+ id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+ title VARCHAR(255),
+ description_reviev TEXT,
+ meal_id INT NOT NULL,
+ stars INT,
+ created_date DATE,
+ FOREIGN KEY (meal_id) REFERENCES Meal(id)
+);
+
+
+ALTER TABLE Meal CONVERT TO CHARACTER SET utf8;
+
+-- Get all meals
+SELECT * FROM Meal;
```

Comment:
Why are you trying to show all rows before you add values?

Reply:
It was a clue for me to see what posts there are. I forgot to comment